CMakeLists.txt

```diff
@@ -196,7 +196,7 @@ function(spvtools_default_compile_options TARGET)
   target_compile_options(${TARGET} PRIVATE ${SPIRV_WARNINGS})
 
   if (${COMPILER_IS_LIKE_GNU})
-    target_compile_options(${TARGET} PRIVATE -std=c++11 -fno-exceptions)
+    target_compile_options(${TARGET} PRIVATE -std=c++17 -fno-exceptions)
```

Comment:
Why do we set the C++ standard directly when we're using `CMAKE_CXX_STANDARD`? Doesn't that take care of adding the flag automatically? And if not, why are we not setting the flag for MSVC?

Reply:
CMAKE_CXX_STANDARD was introduced in Cmake 3.1. https://cmake.org/cmake/help/latest/variable/CMAKE_CXX_STANDARD.html#variable:CMAKE_CXX_STANDARD 

But I updated min CMake version to 3.17, so don't need this flag anymore.  Use CMAKE_CXX_STANDARD

Reply:
Huh.  I think *all* we need is to check CMAKE_CXX_STANDARD, and delete this explicit -std= option?

Reply:
Yes, changed this so we only change/check CMAKE_CXX_STANDARD. Also, disabled extensions so we compile in std++17, not gnu++17.